modules/kernel/src/main/scala/org/tesselation/kernel/Cell.scala

```diff
@@ -0,0 +1,74 @@
+package org.tesselation.kernel
+
+import cats._
+import cats.implicits._
```

Comment:
Please use new syntax here, `import cats.implicits._` should not be used anymore, instead:
`import cats.syntax.all._` or `import cats.syntax.functor._` and so on for every feature you want

Reply:
Right!